utils.js

```diff
@@ -1,6 +1,7 @@
 /* CONSTANTS */
 
 const param = {};
+param.randomTriplets = true;
```

Comment:
Probably shouldn't set this to true on the main branch. Also, could you add a comment describing the purpose of this, just to be incredibly explicit?